src/components/ContactUs.js

```diff
@@ -0,0 +1,66 @@
+function ContactUs(){
+/* Address Object  start */
+const AddressDetails=[
+    {
+    "Email":"info@stackmybiz.com",
+    "Phone":"(+1) 510-761-5117",
+    "Address":{
+          "Street":"5602 Pandorea Terrace,",
+          "City":"Newark, CA, 94560, "
+         },
+    },
+    {
+     "Phone":"(+91) 7050002221",
+     "Address":
+        {
+          "Street":"Honeycomb Infotech,",
+          "City":"Tagore Hill Road,",
+          "County":"Ranchi Jharkhand, India - 834006"
+        }
+    }
+]
+/* Address Object End */
+ return(
+         <div className="Contact Us">
+               <div className=" md:text-xl">
+                  <h1 className="font-bold font-[Poppins] m-1">
+                      Contact Us
+                  </h1>  
+                </div>
+            <div className="Address1 font-[Poppins] ">
+               <ul className="p-2">
+                   <li className="flex">
+                     <svg xmlns="http://www.w3.org/2000/svg" className="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor" strokeWidth={2}>
+                        <path strokeLinecap="round" strokeLinejoin="round" d="M3 8l7.89 5.26a2 2 0 002.22 0L21 8M5 19h14a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v10a2 2 0 002 2z" />
+                    </svg>
+                    <span>&nbsp;{AddressDetails[0].Email}</span>
```

Comment:
We should not refer to a  data like this. What if `AddressDetails[0].Email` does not exist? The page will crash.
In this case we should use either map function or object de-structuring to render the value.
We should also check we the value exists or not using conditional operator.
Eg `AddressDetails[0].Email?AddressDetails[0].Email:null`